File: js/K번째 수.js
function solution(array, commands) {
    const answer = []
    for (let a=0; a<commands.length; a++) {
        let i = commands[a][0];
        let j = commands[a][1];
        let k = commands[a][2];
        let new_arr = array.slice(i-1,j).sort((a,b) => a-b);
        answer.push(new_arr[k-1]);
    }
    return answer;
}

File: python/K번째 수.py
def solution(array, commands):
    answer =[]
    for i in range(len(commands)):
        i,j,k = commands[i][0], commands[i][1], commands[i][2]
        new_arr = sorted(array[i-1:j])
        answer.append(new_arr[k-1])
    return answer